src/waitForAxeResults.ts

```diff
@@ -0,0 +1,28 @@
+import test from '@playwright/test'
+import type { AxeResults, RunOptions } from 'axe-core'
```

Comment:
When @mskelton [removed the return type annotation](https://github.com/Widen/expect-axe-playwright/pull/23/commits/1362f15034ef481d0ef01a296c6cff01e6db635a) from `waitForAxeResults`, this import became obsolete.

```suggestion
import type { RunOptions } from 'axe-core'
```